lib/ability.rb

```diff
@@ -126,6 +128,8 @@ def initialize(user, resource, controller = nil)
         end
 
         can [:administer], User
+        can([:edit, :update], User) { |target_user| target_user.external? }
```

Comment:
I think code climate is suggesting this line looks like `can [:edit, :update], User, &external?`. I'm not sure if I like that any better or worse. Up to you if you want to ignore this warning.